quickstart/deploy-mayastor.md

```diff
@@ -156,7 +156,7 @@ mayastor-csi   3         3         3       3            3           kubernetes.i
 
 ### Control Plane
 
-### Core Agents
+## Core Agents
```

Comment:
The indentation is the wrong way around:  The various control plane components e.g. "Core Agents" should be a sub-ordinate heading, under Control Plane.

i.e. 

## Control Plane
...
### Core Agents
...
### REST
etc.

Reply:
Oh! I mixed that up. Thanks for pointing out. Fixed.